Button/Button.js

```diff
@@ -275,7 +275,7 @@ const ButtonBase = kind({
 			'--agate-button-animation-delay': animationDelay,
 			'--agate-button-badge-bg-color': badgeColor
 		}),
-		minWidth: ({iconOnly, minWidth}) => ((minWidth != null) ? minWidth : !iconOnly)
+		minWidth: ({minWidth}) => ((minWidth != null) ? minWidth : false)
```

Comment:
If converting the `!iconOnly` to `false`, then there is no `min-width` style when using an icon and a text togetherr.

For example. if using a `arrowlargedown` icon and one space (` `), then
the `min-width` style is applied not in Agate Button but in Sandstone Button.

Sandstone
![image](https://user-images.githubusercontent.com/4239873/101856084-f6230700-3ba7-11eb-9707-864589b768e3.png)

Agate
![image](https://user-images.githubusercontent.com/4239873/101856105-01763280-3ba8-11eb-92db-65363aceddc8.png)

So my preference is not fixing the Agate Button but fixing the Key with proper min-width and border-radius.
Please let me know your thought.